docker/README.md

```diff
@@ -22,3 +22,36 @@ docker run ucx-py
 ```
 
 The container above will run UCX-Py tests and benchmarks.
+
+## Infiniband/NVLink-enabled docker file
+
+In addition to the reference Docker image, the `UCXPy-CUDA.dockerfile` builds an
+image with support for CUDA and MOFED. This is based on the
+[nvidia/cuda:11.5.2-devel-ubuntu20.04](https://hub.docker.com/layers/cuda/nvidia/cuda/11.5.2-devel-ubuntu20.04/images/sha256-fed73168f35a44f5ff53d06d61a1c55da7c26e7ca5a543efd78f35d98f29fd4a?context=explore)
+image, and uses MOFED version 5.3-1.0.5.0. To function successfully the OFED
+version reported by `ofed_info -s` on the host should match this version.
+
+To use this image, first build it
+```bash
+docker build -t ucx-py-ib -f UCXPy-CUDA.dockerfile .
+```
+
+### Running
+
+To expose high-performance transports from the host in the container requires a
+number of additional flags when running the container. `docker run --privileged`
```

Comment:
Both sentences actually scan rather badly (since the parenthetical starts and finishes with "the container").

How about:

```suggestion
To expose high-performance transports from the host requires a
number of additional flags when running the container. `docker run --privileged`
```

Reply:
I'm not a native English speaker, but this sentence sounds strange. "To expose high-performance transports from the host requires ...", it seems like we're missing the subject that requires the additional flags. Perhaps we could rephrase it like:

> The container requires a number of additional flags to expose high-performance transports from the host.

Reply:
I guess "To expose high-performance transports from the host requires that one provide a number of additional flags when running the container", but yes, flipping things round will also do it (will fix).

Reply:
Done.